src/dls_bluesky_core/stubs/flyables.py

```diff
@@ -0,0 +1,59 @@
+import bluesky.plan_stubs as bps
+from bluesky.protocols import Flyable
+
+from dls_bluesky_core.core import MsgGenerator, group_uuid
+
+
+def fly_and_collect(
+    flyer: Flyable,
+    flush_period: float = 0.5,
+    checkpoint_every_collect: bool = False,
+    stream_name: str = "primary",
+) -> MsgGenerator:
+    """Repeatedly fly and collect a flyer, flushing with a period.
```

Comment:
Very minor but don't we only fly once? We repeatedly collect but fly?

Reply:
```suggestion
    ""Fly then repeatedly collect a flyer, flushing with a period.
```

?